versioned/mongodb/src/main/java/com/dremio/nessie/versioned/store/mongodb/MongoDBStore.java

```diff
@@ -300,4 +347,19 @@ void resetCollections() {
     }
     return (MongoCollection<T>) collection;
   }
+
+  private boolean shouldUseAggPipeline(UpdateExpression update) {
```

Comment:
NIT: Not sure if shortening Agg adds much, as the function name is already long. Maybe going full length might be more meaningful.